allennlp/training/metrics/drop_em_and_f1.py

```diff
@@ -0,0 +1,73 @@
+import string
+from typing import Tuple, List, Union
+
+from overrides import overrides
+
+from allennlp.tools.squad_eval import metric_max_over_ground_truths
+from allennlp.tools.drop_eval import (get_metrics as drop_em_and_f1,
+                                      answer_json_to_strings)
+from allennlp.training.metrics.metric import Metric
+
+
+STOPWORDS = set(["a", "an", "the"])
+PUNCTUATIONS = set(string.punctuation)
```

Comment:
Looks like these two values are not being used anywhere.

Reply:
Thanks, nice catch.